application/src/main/java/org/thingsboard/server/controller/AuthController.java

```diff
@@ -82,7 +86,13 @@ public class AuthController extends BaseController {
     @GetMapping(value = "/auth/user")
     public User getUser() throws ThingsboardException {
         SecurityUser securityUser = getCurrentUser();
-        return userService.findUserById(securityUser.getTenantId(), securityUser.getId());
+        User user = userService.findUserById(securityUser.getTenantId(), securityUser.getId());
+        if (user.getAdditionalInfo().isObject()) {
+            ObjectNode additionalInfo = (ObjectNode) user.getAdditionalInfo();
+            processDashboardIdFromAdditionalInfo(additionalInfo, DEFAULT_DASHBOARD);
+            processDashboardIdFromAdditionalInfo(additionalInfo, HOME_DASHBOARD);
+        }
```

Comment:
This code is now duplicated three times!
![image](https://github.com/user-attachments/assets/327b7641-3667-4163-b991-69593e460d1f)
![image](https://github.com/user-attachments/assets/37019885-18ca-4732-a7a9-345a4bf37349)